superset-frontend/plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -454,6 +455,41 @@ export default function TableChart<D extends DataRecord = DataRecord>(
         className += ' dt-is-filter';
       }
 
+      type CellClicked = {
+        columnKey: string;
+        rowIndex: number;
+        cellData: DataRecordValue;
+        isAnchor: boolean;
+      };
+
+      const cellClicked = (
+        column: DataColumnMeta,
+        row: Row<D>,
+        value: DataRecordValue,
+      ) => {
+        let isAnchor = false;
+        if (typeof value === 'string') {
+          const parsed = new DOMParser().parseFromString(value, 'text/html');
+          const element = parsed.body.firstChild as HTMLElement;
+          if (element?.tagName === 'A' && element.getAttribute('href') === '#') {
+            const data = element.getAttribute('data');
+            if (data) {
+              try {
+                value = JSON.parse(data);
+                isAnchor = true;
+              } catch (error) {}
```

Comment:
I guess it will be good to inform =>  console.log('data is not a valid object', error)

Reply:
Every message type should be handled differently and if you take my idea to keep a state that updates on emits then that could be it. Anyway, it's a start...